doc/rst/technotes/dsi.rst

```diff
@@ -954,5 +954,173 @@ implicitly for each ``GlobalDomain`` object and so will be local in any case.
 Phase 4: Bulk-Transfer Interface
 =================================
 
-The bulk-transfer interface design is still in flux. Once finalized, it will
-be documented here.
+What is the bulk-transfer interface?
+------------------------------------
+
+While the Chapel language does not require a specific implementation for array
+assignment, in practice the values are assigned individually:
+
+ .. code-block:: chapel
+
+  for (d, s) in zip(dest, src) {
+    d = s;
+  }
+
+When assignment occurs between arrays on different locales, this can result in
+significant communication overhead. The bulk-transfer interface aims at
+reducing communication between locales by enabling domain maps to handle
+certain kinds of assignments themselves.
+
+How does the bulk-transfer interface work with DSI?
+---------------------------------------------------
+
+The bulk-transfer interface is an optional interface supported only on array
+descriptors. The following methods are prefixed with ``doi`` for "Domain map
+Optional Interface." An array descriptor can implement one or more of these
+methods to opt-in to support for bulk-transfers:
+
+.. method:: proc GlobalArray.doiBulkTransferToKnown(myDom:domain, otherClass, otherDom:domain) : bool
```

Comment:
This style does not seem to render properly.  I'm not entirely certain why not, though it might have something to do with the extra space here: `) :`.  That's my hypothesis, though.